backstop/tests/skip-to-content.html

```diff
@@ -177,19 +177,25 @@ <h1>Skip to content link</h1>
           </button>
         </div>
       </header>
-      <div style="border: solid 1px">
-        <p>This is not the main content. Press 'tab' to see skip-to-content-link component. You might need to click on
-          the top of the page or the URL first. Press 'enter' after focusing on skip-to-content-link to skip to the main
+      <div style="padding: 11px 0">
+        <p>This is not the main content. Press&nbsp;<kbd class="iui-keyboard">tab</kbd>&nbsp;to see skip-to-content-link
+          component. You might need to click on
+          the top of the page or the URL first. Press&nbsp;<kbd
+            class="iui-keyboard"
+            title="Return"
+          >&crarr;</kbd>&nbsp;after focusing on skip-to-content-link to skip to the main
           content below.</p>
       </div>
     </section>
-    <br>
+
+    <br />
+
     <section id="demo-skip-to-here">
-      <div style="height: 1000px; border: solid 1px;">
-        <p>This is the main content. <a
+      <div style="height: 1000px; border: solid 1px var(--iui-color-background-border); padding: 11px 12px;">
```

Comment:
Can you move these into the `style` tag?

Reply:
Moved from inline styling into `<style>`.  Was able to remove the styling of the top section completely and replaced with a `<br />`.